flutter_local_notifications/lib/src/platform_specifics/darwin/initialization_settings.dart

```diff
@@ -34,6 +35,11 @@ class DarwinInitializationSettings {
   /// Default value is true.
   final bool requestBadgePermission;
 
+  /// Request permission to send provisional notification for iOS 12+
```

Comment:
Could you add information on the version of macOS that the setting applies to? Another thing is could you follow the existing convention in the API docs has on how to capture the supported OS versions? See `defaultPresentSound` for an example at https://github.com/MaikuB/flutter_local_notifications/blob/558b2ddbfa0c6360e5fbca33300c69e8938e1f7e/flutter_local_notifications/lib/src/platform_specifics/darwin/initialization_settings.dart#L68. What I'm referring to on following so there's consistency is the wording and how this is another paragraph after mentioning the default value

Reply:
Addressed at [82e55c4](https://github.com/MaikuB/flutter_local_notifications/pull/2022/commits/82e55c48d6b4093e014b01afbb3d427ee2f88878)